modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/date/DateDDMFormFieldTemplateContextContributor.java

```diff
@@ -42,7 +42,10 @@
  */
 @Component(
 	immediate = true,
-	property = "ddm.form.field.type.name=" + DDMFormFieldTypeConstants.DATE,
+	property = {
+		"ddm.form.field.type.name=" + DDMFormFieldTypeConstants.DATE,
+		"ddm.form.field.type.name=" + DDMFormFieldTypeConstants.DATE_TIME
+	},
```

Comment:
@rodrigopaulino so we didn’t need to adapt the `DDMFormFieldTypeServicesTracker`? 🎉

Reply:
Correct. I believe I misinterpreted the `openSingleValueMap`/`openMultiValueMap` during our meeting. The difference between those two is that, in case there's more than one component using the same property value, the "SingleValue" will only reference the last one (I assume) and the "MultiValue" will reference a list with all of them. So, since we're adding a new property value, there will be two entries in the map pointing to the same component. Pretty neat. 😛